static/templates/stream_types.hbs

```diff
@@ -1,43 +1,49 @@
+<div class="grey-box">
+    <div class="input-group">
+        <div class="alert stream-privacy-status"></div>
+        <h4>{{t 'Who can access the stream?'}}
+            {{> help_link_widget link="/help/stream-permissions" }}
+        </h4>
+        {{#each stream_privacy_policy_values}}
+        <div class="radio-input-parent">
             <input type="radio" name="privacy" value="{{ this.code }}" {{#if (eq this.code ../stream_privacy_policy) }}checked{{/if}} />
-            <b>{{ this.name }}:</b> {{ this.description }}
-        </label>
-    </li>
-    {{/each}}
-    <h4>{{t 'Who can post to the stream?'}}
-        {{> help_link_widget link="/help/stream-sending-policy" }}
-    </h4>
-    {{#each stream_post_policy_values}}
-    <li>
-        <label class="radio">
+            <label class="radio">
+                <b>{{ this.name }}:</b> {{ this.description }}
+            </label>
+        </div>
+        {{/each}}
+    </div>
+
+    <div class="input-group">
+        <h4>{{t 'Who can post to the stream?'}}
+            {{> help_link_widget link="/help/stream-sending-policy" }}
+        </h4>
+        {{#each stream_post_policy_values}}
+        <div class="radio-input-parent">
             <input type="radio" name="stream-post-policy" value="{{ this.code }}" {{#if (eq this.code ../stream_post_policy) }}checked{{/if}} />
-            {{ this.description }}
-        </label>
-    </li>
-    {{/each}}
+            <label class="radio">
+                {{ this.description }}
+            </label>
+        </div>
+        {{/each}}
+    </div>
+
     {{#if (or is_owner is_stream_edit)}}
-    <h4>{{t "Message retention for stream" }}
-        {{> help_link_widget link="/help/message-retention-policy" }}
-    </h4>
+    <div>
+        <h4>{{t "Message retention for stream" }}
+            {{> help_link_widget link="/help/message-retention-policy" }}
+        </h4>
 
-    {{> settings/upgrade_tip_widget}}
+        {{> settings/upgrade_tip_widget}}
 
-    <li>
-        <div class="input-group inline-block">
+        <div class="input-group inline-block message-retention-setting-group">
             <label for="stream_message_retention_setting" class="dropdown-title">{{t "Message retention period" }}
                 <i class="fa fa-info-circle settings-info-icon realm_allow_message_deleting_tooltip tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Only owners can change stream message retention policy.' }}"></i>
             </label>
             <select name="stream_message_retention_setting"
               class="stream_message_retention_setting" class="prop-element"
               {{#if disable_message_retention_setting}}disabled{{/if}}>
-                <option value="realm_default">{{t 'Use organization level settings' }} {{realm_message_retention_setting}}</option>
+                <option value="realm_default">{{t "Use organization level settings "}}{{realm_message_retention_setting}} </option>
```

Comment:
What's the story with these changes?  A screenshot might be helpful.

In any case, I think the i18n approach taken here seems wrong, since we seem to be relying on word order being like English; I think what we want is `"Use organization level settings ({realm_message_retention_setting})"` as the string we provide to translators, so they can adjust word order.

Reply:
Fixed translation issue in its own separate commit. Also I have modified the commit message to explain more clearly about the reason for html changes in this commit ([https://github.com/zulip/zulip/pull/18872/commits/4eaf45f1ac51aaa62db1e54683bc7838b948e098](https://github.com/zulip/zulip/pull/18872/commits/4eaf45f1ac51aaa62db1e54683bc7838b948e098)).